--- alphafold/data/pipeline.py	2023-11-07 03:38:57.493908530 +0000
+++ alphafold/data/pipeline_org.py	2023-11-06 22:47:39.993496469 +0000
@@ -126,30 +126,21 @@
                uniref_max_hits: int = 10000,
                use_precomputed_msas: bool = False):
     """Initializes the data pipeline."""
-    if "SLURM_NTASKS" in os.environ:
-        ncpus=int(os.getenv("SLURM_NTASKS"))
-    else:
-        ncpus=4
-
     self._use_small_bfd = use_small_bfd
     self.jackhmmer_uniref90_runner = jackhmmer.Jackhmmer(
         binary_path=jackhmmer_binary_path,
-        database_path=uniref90_database_path,
-        n_cpu=ncpus)
+        database_path=uniref90_database_path)
     if use_small_bfd:
       self.jackhmmer_small_bfd_runner = jackhmmer.Jackhmmer(
           binary_path=jackhmmer_binary_path,
-          database_path=small_bfd_database_path,
-          n_cpu=ncpus)
+          database_path=small_bfd_database_path)
     else:
       self.hhblits_bfd_uniref_runner = hhblits.HHBlits(
           binary_path=hhblits_binary_path,
-          databases=[bfd_database_path, uniref30_database_path],
-          n_cpu=ncpus)
+          databases=[bfd_database_path, uniref30_database_path])
     self.jackhmmer_mgnify_runner = jackhmmer.Jackhmmer(
         binary_path=jackhmmer_binary_path,
-        database_path=mgnify_database_path,
-        n_cpu=ncpus)
+        database_path=mgnify_database_path)
     self.template_searcher = template_searcher
     self.template_featurizer = template_featurizer
     self.mgnify_max_hits = mgnify_max_hits
